app/src/main/java/com/metrics_board/app/service/roll/ProjectService.java

```diff
@@ -60,4 +92,32 @@ private ProjectResponse createProjectResponse(Project project) {
                 .updatedAt(project.getUpdatedAt())
                 .build();
     }
+
+    private void validateProjectRequest(ProjectRequest request) {
+        if (request.getName() == null && request.getDescription() == null && request.getStatus() == null) {
+            throw new MissedDataToUpdateException("Missed data to update");
+        }
+
+        if (request.getName() != null && request.getName().equals("")) {
+            throw new MissedDataToUpdateException("Invalid name to update");

```

Comment:
In messages where you referring to some field better to use single quotes like `Invalid 'name' to update`